__tests__/endpoints/app.test.js

```diff
@@ -251,4 +251,33 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(body.msg).toBe("Not found");
       });
   });
-});
\ No newline at end of file
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204, and returns No Content", () => {
+    return request(app).delete("/api/comments/13").expect(204);
+    /* It is not need it, 204 don't send anything
+      
+      .then(({ res }) => {
+        expect(res.statusMessage).toBe("No Content");
+      }); */
+  });
```

Comment:
You are correct, a 204 does not include a body or send anything with it, so the 204 status code is enough!